src/ucp/core/ucp_context.c

```diff
@@ -307,9 +307,10 @@ static ucs_config_field_t ucp_config_table[] = {
    "Experimental: enable new protocol selection logic",
    ucs_offsetof(ucp_config_t, ctx.proto_enable), UCS_CONFIG_TYPE_BOOL},
 
-  {"KEEPALIVE_TIMEOUT", "inf",
-   "Time period between keepalive rounds (\"inf\" - disabled).",
-   ucs_offsetof(ucp_config_t, ctx.keepalive_timeout), UCS_CONFIG_TYPE_TIME_UNITS},
+  /* TODO: set for keepalive more reasonable values */
+  {"KEEPALIVE_TIMEOUT", "0",
```

Comment:
1. What is the plan to address this TODO?
2. maybe rename KEEPALIVE_TIMEOUT to KEEPALIVE_INTERVAL in UCP?

Reply:
1. originally we planned to set any pre-defined value after real runs
2. renamed to KEEPALIVE_INTERVAL